apps/explorer/src/pages/id-page/PageContent.tsx

```diff
@@ -0,0 +1,145 @@
+import { Banner } from '~/ui/Banner';
+import { Divider } from '~/ui/Divider';
+import { FieldsContent } from '~/pages/object-result/views/TokenView';
+import { TabHeader } from '~/ui/Tabs';
+import { ErrorBoundary } from '~/components/error-boundary/ErrorBoundary';
+import { TransactionsForAddressTable } from '~/components/transactions/TransactionsForAddress';
+import TransactionBlocksForAddress from '~/components/TransactionBlocksForAddress';
+import { useBreakpoint } from '~/hooks/useBreakpoint';
+import { OwnedCoins } from '~/components/OwnedCoins';
+import { OwnedObjects } from '~/components/OwnedObjects';
+import { LOCAL_STORAGE_SPLIT_PANE_KEYS, SplitPanes } from '~/ui/SplitPanes';
+import { useSuiClient } from '@mysten/dapp-kit';
+import { useQuery } from '@tanstack/react-query';
+
+const LEFT_RIGHT_PANEL_MIN_SIZE = 30;
+
+function OwnedObjectsSection({ address }: { address: string }) {
+	const isMediumOrAbove = useBreakpoint('md');
+
+	const leftPane = {
+		panel: (
+			<div className="mb-5 h-full md:h-coinsAndAssetsContainer">
+				<OwnedCoins id={address} />
+			</div>
+		),
+		minSize: LEFT_RIGHT_PANEL_MIN_SIZE,
+		defaultSize: LEFT_RIGHT_PANEL_MIN_SIZE,
+	};
+
+	const rightPane = {
+		panel: (
+			<div className="mb-5 h-full md:h-coinsAndAssetsContainer">
+				<OwnedObjects id={address} />
+			</div>
+		),
+		minSize: LEFT_RIGHT_PANEL_MIN_SIZE,
+	};
+
+	return (
+		<TabHeader title="Owned Objects" noGap>
+			<div className="flex h-full flex-col justify-between">
+				<ErrorBoundary>
+					{isMediumOrAbove ? (
+						<SplitPanes
+							autoSaveId={LOCAL_STORAGE_SPLIT_PANE_KEYS.ADDRESS_VIEW_HORIZONTAL}
+							dividerSize="none"
+							splitPanels={[leftPane, rightPane]}
+							direction="horizontal"
+						/>
+					) : (
+						<>
+							{leftPane.panel}
+							<div className="my-8">
+								<Divider />
+							</div>
+							{rightPane.panel}
+						</>
+					)}
+				</ErrorBoundary>
+			</div>
+		</TabHeader>
+	);
+}
+
+function TransactionsSection({ address, isObject }: { address: string; isObject: boolean }) {
+	const client = useSuiClient();
+
+	const {
+		data: transactionsForAddressData,
+		isPending,
+		isError,
+	} = useQuery({
+		queryKey: ['transactions-for-address', address],
+		queryFn: () =>
+			client.queryTransactionBlocks({
+				filter: {
+					FromAndToAddress: {
+						from: address,
+						to: address,
+					},
+				},
+				order: 'descending',
+				limit: 100,
+				options: {
+					showEffects: true,
+					showInput: true,
+				},
+			}),
+		enabled: !isObject,
+	});
+
+	return (
+		<ErrorBoundary>
+			{isObject ? (
+				<TransactionBlocksForAddress address={address} />
+			) : (
+				<TransactionsForAddressTable
+					data={transactionsForAddressData?.data ?? []}
+					isPending={isPending}
+					isError={isError}
+					address={address}
+				/>
+			)}
+		</ErrorBoundary>
+	);
+}
+
+export function PageContent({ address, error }: { address: string; error?: Error | null }) {
```

Comment:
Feels wrong to take this error prop, based on the return I'd expect this to just be at the parent level, conditionally rendering either this or the error box wherever this prop is passed in.